bokehjs/src/coffee/models/formatters/mercator_tick_formatter.coffee

```diff
@@ -0,0 +1,30 @@
+import {BasicTickFormatter} from "./basic_tick_formatter"
+import * as p from "core/properties"
+import {proj4, mercator} from "core/util/proj4"
+
+export class MercatorTickFormatter extends BasicTickFormatter
+  type: 'MercatorTickFormatter'
+
+  @define {
+    dimension: [ p.LatLon ]
+  }
+
+  doFormat: (ticks, loc) ->
+    if ticks.length == 0
+      return []
+
+    if not @dimension?
+      throw Error("MercatorTickFormatter.dimension not configured")
```

Comment:
Interesting, there are other places in the code that do it wrong then. I will update them all.